cif_pow.dic

```diff
@@ -10617,6 +11197,48 @@ save_PD_PROC_OVERALL
     _name.object_id               PD_PROC_OVERALL
     _category_key.name            '_pd_proc_overall.diffractogram_id'
 
+    loop_
+      _description_example.case
+      _description_example.detail
+;
+          _pd_diffractogram.id   PATTERN_42
+
+          _pd_proc.2theta_range_min   5.00
+          _pd_proc.2theta_range_max  95.00
+          _pd_proc.2theta_range_inc   0.02
+          _pd_proc.number_of_points  4501
+
+          loop_
+          _pd_data.id
```

Comment:
This item does not seem to be defined in the current version of the dictionary.

Reply:
Yep. Should be point_id